src/style/shared/dialog.less

```diff
@@ -89,8 +89,8 @@ dialog {
     max-height: 25rem;
     height: auto;
     border-radius: @dialog-radius;
-    .background-transparency(@warm-white, 1);
-    box-shadow: 0 ( @hairline * 2 ) 0 0 @dialog-shadow, 0 -2px 0 0 @warm-black;
+    background:@dark-black;
```

Comment:
@dark-black used here: would we ever want that background to be not dark-black? Should this have a more meaningful color variable name